composer.json

```diff
@@ -26,12 +26,13 @@
         }
     },
     "require": {
-        "php": "^7.0",
+        "php": "^7.0 || ~8.1",
         "symfony/config": "^2.8 || ^3.0 || ^4.0",
         "symfony/console": "^2.8 || ^3.0 || ^4.0",
+        "symfony/http-client": "^5.4",
```

Comment:
```suggestion
```

1. Library doesn't use this dependency
2. You're introducing a breaking change without the need to do that
3. Some dependencies are to be provided by the implementing application
  See https://github.com/prgTW/healthchecks-bundle/blob/509b3fc79d3ffc7a9f5bd498762c7172ea7b3ebd/composer.json#L40-L41